paimon-core/src/main/java/org/apache/paimon/mergetree/LookupLevels.java

```diff
@@ -207,12 +211,8 @@ private LookupFile createLookupFile(DataFileMeta file) throws IOException {
         return new LookupFile(localFile, file, lookupStoreFactory.createReader(localFile, context));
     }
 
-    @Override
-    public void close() throws IOException {
```

Comment:
Should we clean all files when closing?

Reply:
But by default, there is no max size configuration.

Reply:
Evict by the retention time?

Reply:
Before the LookupLevels's cache file will be cleared when the writer is idled (no data written to in the latest snapshot). 

Now, the cache file will still be there after the idle writer closed and evict by the retention time by default.

Reply:
Yes, it is better to remove all files when closing this writer.

Reply:
You can manage a collection in `LookupLevels`.

Reply:
> You can manage a collection in `LookupLevels`.

Thanks, Make sense to me.